remote.go

```diff
@@ -1027,3 +1064,46 @@ func (o *Remote) Prune(callbacks *RemoteCallbacks) error {
 	}
 	return nil
 }
+
+// DefaultApplyOptions returns default options for remote create
+func DefaultRemoteCreateOptions() (*RemoteCreateOptions, error) {
+	runtime.LockOSThread()
+	defer runtime.UnlockOSThread()
+
+	opts := C.git_remote_create_options{}
+	ecode := C.git_remote_create_options_init(&opts, C.GIT_REMOTE_CREATE_OPTIONS_VERSION)
+	if int(ecode) != 0 {
+		return nil, MakeGitError(ecode)
+	}
+
+	return &RemoteCreateOptions{
+		Repository: nil,
+		Flags:      0,
+	}, nil
+}
+
+func remoteCreateOptionsToC(opts *RemoteCreateOptions) (copts *C.git_remote_create_options) {
```

Comment:
could this follow the `populateXxxOptions` convention found in other files? e.g. https://github.com/libgit2/git2go/blob/4b2ac7c998be677d865367908787f17fb570c679/revert.go#L18